### PR TITLE
EMKTG-3153 : Pre monetization messages of module

### PR DESCRIPTION
**Pull request by ClaraLpresta**

### CODE REVIEW

File: _dev/apps/ui/src/components/monetization/pre-monetization-banner-info.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+const localStorageDisplayBannerSuccess = 'banner-success-pre-monatization';
+const localStorageDisplayBannerInfo = 'banner-info-pre-monatization';
+
+export default defineComponent({
+  methods: {
+    onClose() {
+      localStorage.setItem(localStorageDisplayBannerInfo, '0');
+
+      this.$store.dispatch('app/GET_BANNER_INFO_LOCAL_STORAGE_MONETIZATION', 0);
```
Comment (ga-devfront) [inline _dev/apps/ui/src/components/monetization/pre-monetization-banner-info.vue:50]: I think it's a better to use your enum her. Like this, your code is more maintainable.
```js
this.$store.dispatch(`app/${ActionsTypes.GET_BANNER_INFO_LOCAL_STORAGE_MONETIZATION}`, 0);
```

File: _dev/apps/ui/src/store/modules/app/actions.ts [hunk 2]
```diff
@@ -119,4 +118,12 @@ export default {
         });
     }
   },
+
+  async [ActionsTypes.GET_BANNER_SUCCESS_LOCAL_STORAGE_MONETIZATION]({commit}: Context, value) {
```
Comment (ga-devfront) [inline _dev/apps/ui/src/store/modules/app/actions.ts:122]: for me it's not a GET but a SET.
And I think you can update the local storage her and not inside vue scripts. That's avoid duplication code.

File: _dev/apps/ui/src/store/modules/app/actions.ts [hunk 3]
```diff
@@ -119,4 +118,12 @@ export default {
...
+    commit(MutationsTypes.SET_BANNER_SUCCESS_PRE_MONETIZATION, value);
+  },
+
+  async [ActionsTypes.GET_BANNER_INFO_LOCAL_STORAGE_MONETIZATION]({commit}: Context, value) {
```
Comment (ga-devfront) [inline _dev/apps/ui/src/store/modules/app/actions.ts:126]: same comment her

File: _dev/apps/ui/src/views/configuration-page.vue [hunk 4]
```diff
@@ -29,5 +34,12 @@ export default defineComponent({
       return this.$store.getters['accounts/GET_PS_ACCOUNTS_CONTEXT_SHOPS'];
     },
   },
+  mounted() {
+    if (this.$refs.PreMonetizationPopin.$refs.modal) {
```
Comment (ga-devfront) [inline _dev/apps/ui/src/views/configuration-page.vue:38]: why do you don't use your store or local storage her ? And I think you can only use internal .vue logic of each element and not using it in parent.

File: _dev/yarn.lock
Comment (ga-devfront) [inline _dev/yarn.lock:1]: I think you don't have to publish the yarn.lock

**Review by ga-devfront — CHANGES_REQUESTED** [review]

File: .gitignore
[the remark this answers is not in the document]
Comment (ClaraLpresta) [reply .gitignore:1]: Because all these files are generated as soon as I launch the application.

[the remark this answers is not in the document]
Comment (Quetzacoalt91) [reply .gitignore:1]: I'm fine with this change :+1:

**Review by ga-devfront — APPROVED** [review]

**Pull request merged**